src/VoteDelegate.sol

```diff
@@ -15,18 +14,17 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-// VoteDelegate - delegate your vote
-pragma solidity 0.6.12;
+pragma solidity ^0.8.16;
 
-interface TokenLike {
+interface GemLike {
```

Comment:
I know it was like this before, but a bit weird that `approve` has a return value defined, while `transfer` and `transferFrom` do not.
I think we can remove it .

Reply:
Done.